services/digest_utils.py

```
@@ -284,3 +284,46 @@ def enhance_existing_digest_with_thumbnails(self, digest: Dict[str, Any], _targe
             enhanced_packets.append(enhanced_packet)
         
         return enhanced_packets
+
+    def enhance_story_packets_with_thumbnails(self, story_packets: List[Any], target_date: str) -> List[Any]:
```

Ruff (0.12.2)

Redefinition of unused `enhance_story_packets_with_thumbnails` from line 216

(F811)